gulpfile.js

```diff
@@ -30,12 +31,18 @@ gulp.task('copySource', gulpTasks.copy({
   src: `${SOURCE_DIR}/guide/**/*.jsx`,
 }).task);
 
-gulp.task('deploy', gulpTasks.deploy({
-  accessKeyId: process.env.AWS_ACCESS_KEY_ID || 'No ENV',
-  bucketName: process.env.AWS_BUCKET_UI_FRAMEWORK || 'No ENV',
-  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || 'No ENV',
-  src: `${DISTRIBUTION_DIR}/**/*.*`,
-}).task);
+gulp.task('deployToGitHubPages', () => (
+  gulp.src('./dist/**/*')
+    .pipe(ghPages())
+));
+
+gulp.task('deploy', done => {
+  runSequence(
+    'production',
+    'deployToGitHubPages',
+    done
+  );
+});
```

Comment:
What do you mean by reducing flexibility for deployments? This is just a convenience task for deploying the production build, since you can just run the 1 task instead of having to run production, wait for it to complete, and then run a separate task. And you can't mess it up by forgetting to run production first. :)